pkgs/by-name/bi/birdtray/package.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, libsForQt5
+, fetchpatch
+, thunderbird
+}:
+
+stdenv.mkDerivation rec {
+  pname = "birdtray";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "gyunaev";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
_I_ didn't use this. I didn't change anything outside of the context of this PR, as people nit about minimal diffs etc etc.

Reply:
I dont :-)

Reply:
btw I already made this argument months ago in https://github.com/NixOS/nixpkgs/issues/277994#issuecomment-1876291525 and have sent a treewide for fixing [a similar (but in that case, objectively wrong use of pname)](https://github.com/NixOS/nixpkgs/pull/326683) but apparently the first case is controversial, and I'd rather litigate that in a treewide than individual PRs at this point